packages/storefront/src/components/PlaygroundSelect.vue

```diff
@@ -3,7 +3,9 @@
     <select :value="value" :aria-label="name" @input="$emit('input', $event.target.value)">
       <option disabled>{{ `Select ${name}` }}</option>
       <!-- prettier-ignore -->
-      <option v-for="value in values" v-bind:key="value" :value="`${value}`.replace(' (deprecated)', '')">{{ value }}</option>
```

Comment:
``${value}`.replace(' (deprecated)', '')`

Not sure if this is used anywhere but it is not included in the new implementation

Reply:
Checked and updated it 👍🏼